src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -213,6 +215,10 @@ export class Kernel implements IKernel {
                 const stopWatch = new StopWatch();
                 try {
                     try {
+                        await this.populateStartKernelInfoForInteractive(
```

Comment:
THis seems like a weird place to put this. That it should be in the nativeInteractiveWindow class before it calls this startNotebook function.